README.md

```diff
@@ -30,10 +30,9 @@ git
 
 
  key features of the application.
-
```

Comment:
- [ ] AS mentioned by the previous reviewer, you should not merge your pull request until you have an approval on it. This means that any requested change is done on the same branch and pushed. 😅 Merging is the last step of development, and when it's done it means all the changes made in the pull request are correct and ready to be merged with main 👍
##### Please make the following changes on this same pull request and re-submit. 
*  Kindly Give your pull request a professional title. e.g Hello Microverse Project 
* Also, please give brief summary of what you did in the description.
* Please edit your README file appropriately. Kindly [read about Markdown syntax used in github.](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)